MAUI/ProgressBar/Appearance.md

```diff
@@ -0,0 +1,392 @@
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("fbb78a"), Value = 100 });
+this.Content = circularProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %} 
+
+![rangecolors](images/appearance/range-colors.png)
+
+The following code sample demonstrates how to apply gradient transition effect to the range colors in the progress bar.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<progressBar:SfLinearProgressBar Progress="100" >
+    <progressBar:SfLinearProgressBar.GradientStops>
+        <progressBar:ProgressGradientStop Color="#88A0D9EF" Value="0"/>
+        <progressBar:ProgressGradientStop Color="#AA62C1E5" Value="25"/>
+        <progressBar:ProgressGradientStop Color="#DD20A7DB" Value="50"/>
+        <progressBar:ProgressGradientStop Color="#FF1C96C5" Value="75"/>
+    </progressBar:SfLinearProgressBar.GradientStops>
+</progressBar:SfLinearProgressBar>
+
+<progressBar:SfCircularProgressBar Progress="100">
+    <progressBar:SfCircularProgressBar.GradientStops>
+        <progressBar:ProgressGradientStop Color="#88A0D9EF" Value="0"/>
+        <progressBar:ProgressGradientStop Color="#AA62C1E5" Value="25"/>
+        <progressBar:ProgressGradientStop Color="#DD20A7DB" Value="50"/>
+        <progressBar:ProgressGradientStop Color="#FF1C96C5" Value="75"/>
+    </progressBar:SfCircularProgressBar.GradientStops>
+</progressBar:SfCircularProgressBar>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearProgressBar linearProgressBar = new SfLinearProgressBar();
+linearProgressBar.Progress = 100;
+linearProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("88A0D9EF"), Value = 0 });
+linearProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("AA62C1E5"), Value = 25 });
+linearProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("DD20A7DB"), Value = 50 });
+linearProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("FF1C96C5"), Value = 75 });
+this.Content = linearProgressBar;
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar();
+circularProgressBar.Progress = 100;
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("88A0D9EF"), Value = 0 });
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("AA62C1E5"), Value = 25 });
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("DD20A7DB"), Value = 50 });
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("FF1C96C5"), Value = 75 });
+this.Content = circularProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %} 
+
+![gradient](images/appearance/gradient.png)
+
+## Thickness
+
+### Linear progress bar
+
+In the linear progress bar, the height of the track, progress and secondary progress can be customized using the `TrackHeight` and `ProgressHeight` and `SecondaryProgressHeight` properties, respectively.
```

Comment:
In the linear progress bar, the height of the track, progress, and secondary progress can be customized using the `TrackHeight,` `ProgressHeight,` and `SecondaryProgressHeight` properties, respectively.